package.json

```diff
@@ -120,19 +108,19 @@
     "eslint-plugin-react": "7.23.2",
     "eslint-plugin-react-hooks": "4.2.0",
     "execa": "5.0.0",
-    "husky": "3.1.0",
+    "husky": "6.0.0",
     "jest": "26.6.3",
     "js-yaml": "3.14.1",
     "lerna": "4.0.0",
-    "lint-staged": "8.2.1",
+    "lint-staged": "10.5.4",
     "lodash": "4.17.21",
     "markdownlint": "0.21.0",
     "npm-run-all": "4.1.5",
     "ora": "5.4.0",
     "prettier": "2.2.1",
     "prismjs": "1.23.0",
     "shx": "0.3.3",
-    "sinon": "7.4.1",
+    "sinon": "^10.0.0",
```

Comment:
Since you are pinning everything else please pin this too

Reply:
Haha, I saw that and then thought "No, Mrugesh said leave it to the bot".

Reply:
There we go, all pinned.